python/lsst/ap/association/packageAlerts.py

```diff
@@ -410,6 +411,8 @@ def createCcdDataCutout(self, image, skyCenter, extent, photoCalib, srcId):
                     % srcId)
             return None
 
+        cutoutPsf = image.psf.computeKernelImage(point).array
```

Comment:
Maybe a comment here as to why this is `computeKernelImage` instead of `computeImage`? I'm still not convinced as to which is correct for this use case.

Reply:
I'm convinced by [previous discussion](https://lsstc.slack.com/archives/C7ZGQAF25/p1698172795741629) that we want the PSF centered in the array.